src/common/makeGraphqlIterator.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable require-atomic-updates */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable immutable/no-let */
+
+import { getOctokit } from '@actions/github';
+
+const MAX_PAGE_SIZE = 100;
+
+export interface IterableList<Iterable> {
+  edges: Array<{
+    node: Iterable;
+  }>;
+  pageInfo: {
+    endCursor: string;
+    hasNextPage: boolean;
+  };
+}
+
+export const makeGraphqlIterator = <ResponseData, IterableData>(
```

Comment:
What was the reason to choose manual creation based on symbols instead of actual `export const makeGraphqlIterator = async function*`?

Reply:
Ah, that's been a long time. Forgot that it existed. Changed :).